### PR TITLE
adding run_query_and_insert_to_table_in_chunks, TableForTest, and tests

### PR DESCRIPTION
**Pull request by tkalir**

### CODE REVIEW

File: anyway/utilities.py [hunk 1]
```diff
@@ -173,6 +176,29 @@ def truncate_tables(db, tables):
...
+    query = order_query_by_columns(base_query, primary_keys)
+    while items_read == chunk_size:
+        chunk = query \
+            .slice(offset, offset + chunk_size) \
+            .all()
+        items_read = len(chunk)
+        offset += items_read
+        yield [row._asdict() for row in chunk]
+
+def run_query_and_insert_to_table_in_chunks(query, table_inserted_to, columns_to_order_query_uniqely, chunk_size, session):
```
Comment (BarVolunteering) [inline anyway/utilities.py:197]: Where is it being used?

**Review by BarVolunteering — COMMENTED** [review]
The code looks good (maybe add type-hinting?), but i am not familiar with the use-case - where/when this logic will run?
@tkalir

**Comment by tkalir** [comment]
> The code looks good (maybe add type-hinting?), but i am not familiar with the use-case - where/when this logic will run? @tkalir

thanks for the comment. it was supposed to be used for updating the CBS tables, but I'm scrapping this PR and it's replaced by this one
https://github.com/hasadna/anyway/pull/2247

**Comment by tkalir** [comment]
replaced by https://github.com/hasadna/anyway/pull/2247

**Pull request closed without merging**